core/src/main/resources/org/apache/spark/ui/static/webui.css

```diff
@@ -88,12 +88,14 @@ a:not([href]):hover {
   height: 50px;
   padding: 10px 15px 10px;
   line-height: 2;
+  white-space: nowrap;
 }
 
 .navbar .navbar-nav .nav-item.active .nav-link {
   background-color: #e5e5e5;
   box-shadow: inset 0 3px 8px rgba(0, 0, 0, 0.125);
   color: #555555;
+  white-space: nowrap;
```

Comment:
I piggyback one CSS change. Because `SQL / DataFrame` contains whitespace, we should make sure this line is separated (or wrapped) to the next line.

Before:

<img width="769" alt="Screen Shot 2022-03-29 at 12 14 16 PM" src="https://user-images.githubusercontent.com/6477701/160525974-73133494-4710-4988-8281-3642894a3cc9.png">

After:

<img width="787" alt="Screen Shot 2022-03-29 at 12 13 18 PM" src="https://user-images.githubusercontent.com/6477701/160525982-19118d58-d8a1-494c-924b-816a28b8c0f9.png">